app/views/profiles/show.html.slim

```diff
@@ -34,10 +34,8 @@
               - else
                 i.glyphicon.glyphicon-remove.text-danger
                 span.sr-only Doesn't work on #{day.to_s}s
-.row
-  .col-xs-12
-    = link_to t('edit_profile'), edit_profile_path(@profile), class: 'btn btn-default'
 
+= form_navigation_btn :edit
 .row
```

Comment:
not without refactoring the helper. it uses non standard text and a target path that is not related to its views controller.